### ISSUE TITLE
 getPcodeOps(Address refAddr) method returns NULL because hfunction is NULL

### ISSUE THREAD

**Issue by Yash0day** [NationalSecurityAgency/ghidra#4277]
I have been using a small python script to access the pcodes of each function using Ghidra scripting engine. The script works fine for small binaries(I think) but doesn't work for large binaries. It is because the hfunction returns NULL resulting in error(Screenshot below). 
Ghidra Github: https://github.com/NationalSecurityAgency/ghidra/blob/master/Ghidra/Features/Decompiler/ghidra_scripts/ShowCCallsScript.java 
![image](https://user-images.githubusercontent.com/24827905/169787889-702798c8-fe59-4449-832f-c7e1268a7b55.png)

This error comes after the _hfunction ==_ None as given below: 
![image](https://user-images.githubusercontent.com/24827905/169788927-f6487c6f-a746-4029-9ccf-1bc7573d8fc5.png)

However, this was not happening with small binaries. I have tested this is 750 KB binaries where it was working fine. Whereas for 1350 KB, I was getting the error. (I can be wrong). 

The python script along with both kind of binaries are kept in the ghidra repo given below. The large file is the one getting error whereas the smaller one gives a proper output with the python script. 
https://github.com/Yash0day/Ghidrapython

I have tried increasing the Decompiler Timeout from 30 to 240 and still having the error.

**Comment by mumbel** [comment]
no, your high_func is None, if hFunction was null in the snippet above it would exit early.

**Comment by Yash0day** [comment]
Do you know why is that happening? 
`high = res.getHighFunction()` from my piece of .py code gives NONE. Hence, getHighFunction() is faulty.
